elasticsearch/templates/service.yaml

```diff
@@ -2,7 +2,15 @@
 kind: Service
 apiVersion: v1
 metadata:
+<<<<<<< HEAD
```

Comment:
It seems that you committed a merge conflict by mistake

Reply:
Oops! Fixed. Thanks!